frontend/libs/studio-pure-functions/src/StringUtils/StringUtils.test.ts

```diff
@@ -104,4 +104,16 @@ describe('StringUtils', () => {
       expect(StringUtils.substringAfterLast('abc/def/', '/')).toBe('');
     });
   });
+
+  describe('capitalize', () => {
+    it('Capitalizes the first letter of the string', () => {
+      expect(StringUtils.capitalize('abc')).toBe('Abc');
+      expect(StringUtils.capitalize('a')).toBe('A');
+      expect(StringUtils.capitalize('')).toBe('');
```

Comment:
The test is a duplicate of the test below, we don't need both. I think we can remove this one 
```suggestion
```